### DISCUSSION TITLE
Whitebark pine

### DISCUSSION THREAD

**Discussion by diazrenata** [mekevans/forestTIME#20] [General]
@mekevans What's the species code for whitebark pine?

**Comment by mekevans** [comment]
101

**Comment by mekevans** [comment]
@cmartinez1997 Here's the whitebark pine discussion thread that we can use to communicate

**Comment by mekevans** [comment]
@diazrenata Ceci and I were wondering if she could join our meeting this Fri at 10 am to talk through duplicate trees that are showing up in the WBP data set...mysterious!

**Reply by cmartinez1997** [reply]
To elaborate a bit more, they are trees censused in the same year, but with different PLT_CNs

**Comment by mekevans** [comment]
We're looking for these additional variables: at the COND level BALIVE, SICOND, SISP, SIBASE, SDI

**Reply by cmartinez1997** [reply]
More condition level variables

DSTRBCD1 Disturbance code 1
DSTRBYR1 Disturbance year 1 
DSTRBCD2 Disturbance code 2 
DSTRBYR2 Disturbance year 2 
DSTRBCD3 Disturbance code 3 
DSTRBYR3 Disturbance year 3

**Reply by diazrenata** [reply]
I can get the other columns, but there isn't an "SDI" variable. There is "SDI_RMRS", "SDIMAX_RMRS", and "SDIPCT_RMRS" which report SDI information for trees from the Rocky Mountain Research Station. Is this what we're looking for?

**Reply by cmartinez1997** [reply]
SDI_RMRS is what we're looking for

**Reply by cmartinez1997** [reply]
whoops why don't we include both SDIMAX_RMRS, and SDI_RMRS. Also if you are able to, additional condition level variables: SLOPE, ASPECT, and CONDPROP_UNADJ

**Reply by cmartinez1997** [reply]
And then at the tree level for recruitment: RECONCILECD (which is for ingrowth)

**Comment by mekevans** [comment]
Renata is going to set up a Box folder that Ceci can connect to and download parts of

**Comment by diazrenata** [comment]
Hi @cmartinez1997! I just pushed an updated version of the whitebark_pine.csv dataset, and a document with instructions for how you can (hopefully!) query the tables on your own computer. The document is here: https://github.com/mekevans/forestTIME/blob/main/use_cases/whitebark_pine/WhitebarkPine_db.md and the .csv is here: https://github.com/mekevans/forestTIME/blob/main/use_cases/whitebark_pine/whitebark_pine.csv

You'll need to download a (300MB) database, which is on Box (download link: https://arizona.box.com/s/c6851saedilk8wic0o4z452i365q80c7) - hopefully that will be OK for your computer? More notes are in the document!

Edit to add: I changed direction from the "synchronized box folder" plan I hatched during our meeting. This new approach is using a lightweight database called DuckDB, which runs in R using the R package "duckdb". The `query_tables` function pulls the data in from the database, and from there you can interact with it just like any old dataframe.

**Reply by cmartinez1997** [reply]
Thanks so much, just getting around to looking at this today. Will let me know if I have any questions!!
